account.py

```diff
@@ -0,0 +1,16 @@
+class Account:
+
+    def __init__(self,id,email,access_token,refresh_token):
+        self.id = id
+        self.email = email
+        self.access_token = access_token
+        self.refresh_token = refresh_token
+    
+    def save(self,database):
+        database.cursor.execute(
+        "INSERT INTO accounts (email,google_access_token,google_refresh_token) VALUES (%s,%s,%s)",
```

Comment:
@HadarGreinsmark and I thinks this should be moved somewhere else as this makes the Account class tightly coupled with the current database